fishtown-analytics/docs.getdbt.com
# Adjust discourse function

## Description & motivation
This adjusts the Discourse Netlify function to return a `200` status rather than `500` if no topics are returned from the Discourse Search API. This is to prevent triggering numerous Datadog alerts. 

I adjusted the `DiscourseHelpFeed` component in the preview link below to search for a tag which doesn't exist in Discourse.

## Testing with the preview

1. Open the [Netlify function log](https://app.netlify.com/sites/docs-getdbt-com/functions/get-discourse-topics?scope=deploypreview:2564) for this preview.
2. Open the [preview link](https://deploy-preview-2564--docs-getdbt-com.netlify.app/docs/build/jinja-macros), this will run the Discourse function
3. View the function log and verify the `Unable to get results from api request` message appears followed by the search query, and a green "Duration" message, meaning the function returned a 200 status.

## To-do before merge
- [ ] Remove edits for Jinja page
